tests/functional/test_color.py

```diff
@@ -36,5 +36,6 @@ def test_context_returns_null_from_null_pointer(self):
 
     def test_equal(self):
         a = Color(0, 0, 0)
-        with self.assertRaises(NotImplementedError):
-            assert a == 1
+        b = Color(1, 1, 1)
+        assert a != b
+        assert a != 1
```

Comment:
Increase the code coverage a bit by testing the `from_pointer` method.

```suggestion
        assert a != 1
        
    def test_from_pointer(self):
        a = Color(1, 2, 3)
        b = Color.from_pointer(a.pointer)
        assert a == b
```